### PR TITLE
[en]: update storage/hwameistor/faqs

### PR DESCRIPTION
**Pull request by WANG0608GitHub**
Update after #5511

### CODE REVIEW

**Comment by WANG0608GitHub** [comment]
see preview:
https://deploy-preview-5516--daocloud-docs.netlify.app/en/storage/hwameistor/faqs

File: docs/en/docs/storage/hwameistor/faqs.md [hunk 1]
```diff
@@ -1,49 +1,66 @@
...
 HwameiStor recommends using StatefulSet for applications with multiple replicas.
 
-StatefulSet deploys the replicated replicas to the same worker node, but creates a corresponding PV volume for each Pod replica. If you need to deploy replicas to different nodes to distribute the workload, you need to manually configure pod affinity.
+StatefulSet deploys the replicated replicas to the same worker node, but creates a proper PV volume for each pod
+replica. If you need to deploy replicas to different nodes to distribute the workload, you need to manually configure using `podaffinity` .
 
 ![img](https://docs.daocloud.io/daocloud-docs-images/docs/storage/hwameistor/img/clip_image004.png)
 
-Deployment cannot share block volumes, so it is recommended to use a single replica.
+Deployment cannot share Block volumes, so it is recommended to use a single replica.
```
Comment (windsonsea) [inline docs/en/docs/storage/hwameistor/faqs.md:30]:
```suggestion
Deployment cannot share block volumes, so it is recommended to use a single replica.
```
这个首字母没必要大写

File: docs/en/docs/storage/hwameistor/faqs.md [hunk 2]
```diff
@@ -1,49 +1,66 @@
...
+The HwameiStor scheduler is deployed as a pod in the HwameiStor namespace.
 
 ![img](https://docs.daocloud.io/daocloud-docs-images/docs/storage/hwameistor/img/clip_image002.png)
 
-When an application (Deployment or StatefulSet) is created, the Pods of the application are automatically deployed to worker nodes that have been configured with HwameiStor local storage capability.
+When an application (Deployment or StatefulSet) is created, the pods of the application are automatically deployed
+to worker nodes that have been configured with HwameiStor local storage capability.
 
-**Question 2: How does HwameiStor handle scheduling for applications with multiple replicas? How is it different from traditional shared storage?**
+## How does HwameiStor handle scheduling for applications with multiple replicas? How is it different from traditional shared storage?
```
Comment (windsonsea) [inline docs/en/docs/storage/hwameistor/faqs.md:21]:
```suggestion
## How does HwameiStor handle scheduling for applications with multiple replicas?

Some users may want to understand the scheduling methods used for applications with replicas and how they differ from traditional shared storage systems.
```
英文这个 heading 太长了，要想想办法

**Review by windsonsea — APPROVED** [review]
See preview:
https://deploy-preview-5516--daocloud-docs.netlify.app/en/storage/hwameistor/faqs

**Pull request merged**